src/System.Data.SqlClient/tests/ManualTests/SQL/ParameterTest/SqlVariantParam.cs

```diff
@@ -123,83 +122,85 @@ private static void SendVariantBulkCopy(object paramValue, string expectedTypeNa
                 {
                     connBulk.Open();
 
-                    // Drop and re-create target bulk copy table.
-                    xsql(connBulk, "if exists(select 1 from sys.tables where name='{0}') begin drop table {0} end", bulkCopyTableName);
-                    xsql(connBulk, "create table {0} (f1 sql_variant)", bulkCopyTableName);
-
-                    // Perform bulk copy to target.
-                    using (SqlBulkCopy bulkCopy = new SqlBulkCopy(connBulk))
+                    ExecuteSQL(connBulk, "create table {0} (f1 sql_variant)", bulkCopyTableName);
+                    try
```

Comment:
https://github.com/dotnet/corefx/pull/20790/files?w=1
Try looking at the diff with this. You will get rid of all the whitespaces.